scrapers/base.py

```diff
@@ -43,3 +44,24 @@ def get_html_content(self, url):
             print "Url is not reachable. Error logged on: {}".format(
                 strftime("%Y-%m-%d %H:%M:%S", gmtime()))
             log.info(str(err))
+
+    def local_store(self, data):
+        """Writes content from the website to file.
+        Usage::
+             the file name to write the data
+        :param_train_data: the filename
+        :rtype: outputs data to file.
+        """
+        file_name = "news"
+        if file_name:
+            with open(os.path.dirname(__file__) +"/data/"+"news"+".json", "r+") as output_file:
+                try:
+                    existing_data = json.load(output_file)
+                except:
+                    existing_data = {'news': []}
+                finally:
+                    if type(existing_data) is not dict:
+                        existing_data = {'news': []}
+                existing_data['news'] += data
```

Comment:
Hmm.. I've we're already going to overwrite the file, is there a need to actually open and check its contents? Should we not just overwrite it?

Reply:
The reason I did this is because every scraper writes its data to the news file. If I don't check the content, it will override the file content and only have data from one scraper.

Reply:
Would that mean the `news.json` file is appended to indefinitely with each scrape? When do we clear the data to ensure that it remains only 7 news stories?

Reply:
Initially I had thought of appending but i thought writing to the file was the most appropriate way of doing it. I truncate the file before writing to it which is handled in the nation scraper.

Reply:
Ok, we should add documentation in the code as to why we are reading the file even though we will still be running a scrape that will remove this same data.